### DISCUSSION TITLE
how to get access to LinkedIn learning course which is partnered with GitHub it is not free for everyone on LinkedIn learning

### DISCUSSION THREAD

**Discussion by RavalSahil1311** [community/community#140784] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

[THE LINK OF THE COURSE ABOUT WHICH I'M ASKING](https://www.linkedin.com/learning/paths/prepare-for-the-github-foundations-certification)

Does anyone know how can I get it ?

**Comment by ebndev** [comment]
Hi @RavalSahil1311, thanks for being a part of the GitHub Community!

This isn't the same course, but here's the [GitHub Foundations Learning Path](https://learn.microsoft.com/en-us/collections/o1njfe825p602p) created by GitHub Training.

You can view information about all certifications on our [Certifications page](https://resources.github.com/learn/certifications/) as well.

Best of luck!
